helmfile/values/falco/falco-common.yaml.gotmpl

```diff
@@ -54,263 +54,45 @@ falcoctl:
       enabled: false
 
 customRules:
-  {{- if .Values.falco.customRules }}
-    {{ toYaml .Values.falco.customRules | nindent 2}}
-  {{- end }}
-  ssh-trafic.yaml: |-
-    - rule: Inbound SSH Connection
-      desc: Detect Inbound SSH Connection
-      condition: >
-        ((evt.type in (accept,listen) and evt.dir=<) or
-          (evt.type in (recvfrom,recvmsg))) and ssh_port
-      output: >
-        Inbound SSH connection (user=%user.name client_ip=%fd.cip client_port=%fd.cport server_ip=%fd.sip)
-      priority: WARNING
-      tags: [network]
-    - rule: Outbound SSH Connection
-      desc: Detect Outbound SSH Connection
-      condition: >
-        ((evt.type = connect and evt.dir=<) or
-          (evt.type in (sendto,sendmsg))) and ssh_port
-        and not (k8s.ns.name = argocd-system and k8s.pod.name startswith argocd-repo-server)
-      output: >
-        Outbound SSH connection (user=%user.name server_ip=%fd.sip server_port=%fd.sport client_ip=%fd.cip)
-      priority: WARNING
-      tags: [network]
   overwrites.yaml: |-
+    - list: trusted_image_repositories
```

Comment:
nit: maybe rename this to something that makes it a little more clear that it belongs to the `k8s_containers` macro.

Reply:
This was intentionally left a bit agnostic since it may be used in other exceptions later. (When working with the old rules this list was used in multiple exceptions, since we are looking to enable them later we chose to leave it as is)

Reply:
Ah, ok. Then I guess it's fine.
But then I think I would like a comment just above that list which informs about which rules this should apply to. So that you know what happens if you add an image to the list.